test/block/blockdownload.test.js

```diff
@@ -1049,13 +1049,7 @@ describe('Block Download', function () {
             assert.equal(events.filestart[0].fileSize, 15);
             assert.equal(events.filestart[1].fileSize, 10);
             assert.equal(events.fileprogress.length, 2);
-            assert.equal(events.fileprogress[0].fileSize, 15);
```

Comment:
No order guarantee anymore

Reply:
Why is that, did that change somewhere?

Even if there is no order guarantee, I think the test should check if either `fileprogress[0]` or `fileprogress[1]` match the expected values.